setup.py

```diff
@@ -12,6 +13,9 @@
     version = fh.readlines()[-1].split()[-1].strip("\"'")
 
 
+if sys.version_info < (3, 5):
```

Comment:
We could add some more dependency checks here to be sure, for example cython and numpy or add them to `install_requires`. I would prefer the first to have one single way of installing this package.

Reply:
That these packages must be installed prior to installation. Then there's one less way to install SMAC3 (namely to not have those packages installed yet).